docs/cloud/cloud-databases/cloud-db-shape.md

```diff
@@ -21,20 +21,23 @@ Cloud Database shapes incur an hourly fee, chargeable at the end of the month.
 {% include /cloud/cloud-before-begin.md %}
 * {% include contact-support.md %} to upgrade your account to access production database shapes
 
-### Trial account and Development shapes
 
-Two database shapes are available on all accounts. These are intended for development and testing purposes.
+### Database Shapes
 
-{% include /cloud-db/cloud-db-shape-dev.md %}
```

Comment:
FYI - As a rule, include files are used in multiple target pages in the site (sorry if that's a "Captain Obvious" statement) 0_0

With that in mind, it's good practice to search for the include file you're editing to work out if it'll affect target pages in unexpected ways.

Let us know if you find an include file that's only got one page target because it could be:
* a hangover from earlier content and a second target page has been archived
* intended for future PRs (e.g., the Ingest flags)